Gemfile.lock

```diff
@@ -20,8 +20,18 @@ GEM
       tzinfo (~> 1.1)
     addressable (2.5.2)
       public_suffix (>= 2.0.2, < 4.0)
+    ansi (1.5.0)
```

Comment:
Should Gemfile.lock be committed or in .gitignore?

Reply:
Normally with a rails or Sinatra app, you would want the Gemfile.lock in the repo, however since we are building a gem it should probably be in the .gitignore.